app/src/main/res/layout-land/onboarding_slide_final.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.onboarding.OnboardingSlideFinalViewModel" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="horizontal">
+
+    <ImageView
+      android:id="@+id/slide_image_view"
+      android:layout_width="0dp"
+      android:layout_height="0dp"
+      android:adjustViewBounds="true"
+      android:scaleType="fitXY"
+      android:src="@drawable/ic_landscape_onboarding_3"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintDimensionRatio="4:4"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+      android:id="@+id/slide_title_text_view"
+      android:layout_width="0dp"
+      android:layout_height="wrap_content"
+      android:layout_marginTop="80dp"
+      android:layout_marginEnd="28dp"
+      android:fontFamily="sans-serif-medium"
+      android:gravity="center_horizontal"
+      android:text="@string/onboarding_slide_3_title"
+      android:textColor="@color/oppiaPrimaryText"
+      android:textSize="24sp"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintHorizontal_bias="1.0"
+      app:layout_constraintStart_toEndOf="@+id/slide_image_view"
+      app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+      android:id="@+id/slide_description_text_view"
+      android:layout_width="0dp"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="10dp"
+      android:layout_marginTop="16dp"
+      android:layout_marginEnd="40dp"
+      android:fontFamily="sans-serif"
+      android:gravity="center_horizontal"
+      android:text="@string/onboarding_slide_3_description"
+      android:textColor="@color/oppiaPrimaryText"
+      android:textSize="16sp"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintHorizontal_bias="0.0"
+      app:layout_constraintStart_toEndOf="@+id/slide_image_view"
+      app:layout_constraintTop_toBottomOf="@+id/slide_title_text_view" />
+
+    <Button
+      android:id="@+id/get_started_button"
+      android:layout_width="240dp"
```

Comment:
This should be `match_parent` as you already have margin to take of all the things.